belvedere/src/main/java/zendesk/belvedere/ImageStreamPresenter.java

```diff
@@ -1,11 +1,23 @@
 package zendesk.belvedere;
 
 
+import static zendesk.belvedere.BelvedereUi.FIVE_SECONDS_DELAY;
+import android.Manifest.permission;
+import android.app.Activity;
+import android.os.Build;
+import android.os.Build.VERSION_CODES;
 import android.view.View;
-
+import android.view.View.OnClickListener;
+import android.view.ViewGroup;
+import androidx.annotation.RequiresApi;
+import java.lang.ref.WeakReference;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
-
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Objects;
+import zendesk.belvedere.PermissionManager.InternalPermissionCallback;
 import zendesk.belvedere.ui.R;
 
 class ImageStreamPresenter implements ImageStreamMvp.Presenter {
```

Comment:
Do we need to update the unit tests for these changes?

Reply:
I don't think we need to update but add an additional one. I'll double check this. Thanks Ian!

Reply:
Hey @IanArb . I doubled check this. I should be adding UI tests for these changes, but this repo doesn't have any UI tests so I'll be skipping this for now.